jsf/injection/src/main/java/org/jboss/as/jsf/injection/weld/legacy/ForwardingApplication.java

```diff
@@ -57,7 +57,7 @@
 /**
  * @author pmuir
  *
- * Bring this class back to allow JSF 1.2 to be used with WildFly
+ * Bring this class back to allow Faces 1.2 to be used with WildFly
```

Comment:
@fjuma what do you recommend here for the `Faces 1.2` reference?  Updating to `Jakarta Server Faces 3.0` does seem to fit exactly.

Reply:
I think we should leave this as is. We should probably remove this class at some point as I don't think it is used anymore.